DiyBlog/settings.py

```diff
@@ -38,6 +46,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'Blog.apps.BlogConfig',
+    'social_django',
```

Comment:
This is just a note: what Python package (Django add-on) does someone who wants to boot this project need to install? We will solve this in a separate issue.